proposals/0255-Enhance-BodyInformation-vehicle-data.md

```diff
@@ -16,118 +16,161 @@ In order to partner with more diverse app partners, we need to provide additiona
 ## Proposed Solution 
 
 Following vehicle data params are added to `BodyInformation` struct:
-* `trunkAjar`
-* `hoodAjar`
-* `frontLeftDoorLocked`
-* `frontRightDoorLocked`
-* `rearLeftDoorLocked`
-* `rearRightDoorLocked`
-* `trunkLocked`
+* `doorsStatus`
+* `gatesStatus`
```

Comment:
```suggestion
* `gateStatus`
```

Reply:
Param name is `gatesStatus` for multiple gates as it is an array.